includes/class-wc-amazon-payments-advanced-rest-api-controller.php

```diff
@@ -384,16 +383,18 @@ protected function authorize_order_v1( $order_id, $authorize_args = array() ) {
 			return $resp;
 		}
 
+		$or = wc_get_order( $order_id );
```

Comment:
Can we give this variable a better name, please?

Reply:
done!